src/Agent/NewRelic.Api.Agent/NewRelic.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Is this something that we need to ILRepack? We should probably test this on a simple console app that doesn't have many dependencies and see that the agent still works as expected.

Reply:
Good point. I'll rework so that all exclusions are in the `UnitTests.runsettings` file instead.